changelogs/fragments/docker-migration-removal.yml

```diff
@@ -0,0 +1,14 @@
+removed_features:
+- >
+  All ``docker`` modules and plugins have been removed from this collection.
+  They have been migrated to the `community.docker <https://galaxy.ansible.com/community/docker>`_ collection.
+  If you use ansible-base 2.10 or newer, redirections have been provided.
+
+  If you use Ansible 2.9 and installed this collection, you need to adjust the FQCNs (``community.general.docker_container`` → ``community.docker.docker_container``) and make sure to install the community.docker collection.
+breaking_changes:
+- >
+  If you use Ansible 2.9 and the ``docker`` plugins or modules from this collections, community.general 2.0.0 results in errors when trying to use the docker content by FQCN, like ``community.general.docker_container``.
+  Since Ansible 2.9 is not able to use redirections, you will have to adjust your playbooks and roles manually to use the new FQCNs (``community.docker.docker_container`` for the previous example) and to make sure that you have ``community.docker`` installed.
+
+  If you use ansible-base 2.10 or newer and did not install Ansible 2.11, but installed (and/or upgraded) community.general manually, you need to make sure to also install ``community.docker`` if you are using any of the ``docker`` plugins or modules.
```

Comment:
Did not install Ansible 2.11 ??  Did you mean 2.10?

Reply:
The first Ansible version this will be contained in is actually 2.11 :)

Reply:
(And now it's 3.0.0 and no longer 2.11...)